code/style.css

```diff
@@ -1,7 +1,431 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+
+/* MOBILE FIRST APPROACH */
+
+body {
+  margin: 0;
+  background-color: #faf4edee;
+  font-family: "didot";
+}
+
+.herovideo video {
+  max-width:100%;
+}
```

Comment:
Consider adding a height if you want to display only the text (if the video is not supported) as the text falls between the navbar and the form right now:
![Skärmavbild 2020-09-07 kl  18 11 30](https://user-images.githubusercontent.com/51529201/92405298-8f6bf880-f135-11ea-8292-1a4ad4874b38.png)